templates/core/Resources/_ti/extensions/String.js

```diff
@@ -0,0 +1,80 @@
+/**
```

Comment:
We don't need this anymore due to https://github.com/appcelerator/node-titanium-sdk/pull/32

Unless this is for example purposes?